### PR TITLE
codestyle: Define RuntimePolicyType and use it

### PR DESCRIPTION
**Pull request by stefanberger**

### CODE REVIEW

File: keylime/types.py [hunk 1]
```diff
@@ -1,9 +1,48 @@
+import sys
 import typing
 
+from keylime.ima.ima_dm import Policies
```
Comment (stefanberger) [inline keylime/types.py:4]: The inclusion of this here could cause some issues down the road. Maybe the types would have to be moved into this file here or into keylime.ima.ima_dm_types.

Comment (THS-on) [reply keylime/types.py:4]: Maybe we can put all the IMA related types into `keylime.ima.types` and keep only the more general ones in `keylime.types`?

Comment (stefanberger) [reply keylime/types.py]: I moved it now.

**Review by THS-on — APPROVED** [review]
LGTM

**Pull request merged**